hooks/commits.php

```diff
@@ -5,14 +5,15 @@
 
 error_reporting(E_ALL);
 
-define('PMAHOOKS', True);
+define('PMAHOOKS', true);
 
 require_once('./lib/github.php');
 
 github_verify_post();
 
 $contributing_url = 'https://github.com/phpmyadmin/phpmyadmin/blob/master/CONTRIBUTING.md';
 $guidelines_url = 'https://github.com/phpmyadmin/phpmyadmin/wiki/Developer_guidelines';
+$dco_link = 'https://github.com/phpmyadmin/phpmyadmin/blob/master/DCO#L39';
```

Comment:
Possibly you didn't mean to include the `#L39` in the URL

Reply:
@ibennetch I finally removed it :)